docs/troubleshooting.md

```diff
@@ -31,26 +31,24 @@ Check below how to get the logs for most used clients:
 
 </details>
 
-An alternative is to pass `--trace` to clojure-lsp during process start and clojure-lsp will log the communication to the server-log.
+An alternative is to pass `--trace-level verbose` to clojure-lsp during process start and clojure-lsp will log the communication to the server-log.
```

Comment:
I might be misunderstanding what you mean by "process start", but when I run this I get an error:

```
$ clojure-lsp --version
clojure-lsp 2022.10.05-16.39.51
clj-kondo 2022.10.05

$ clojure-lsp --trace-level verbose
The following errors occurred while parsing your command:

Unknown option: "--trace-level"
```

Reply:
@pfernandez this will only be available on next release, you can try a clojure-lsp nightly build here meanwhile: https://github.com/clojure-lsp/clojure-lsp-dev-builds